build.gradle.kts

```diff
@@ -33,15 +34,29 @@ repositories {
     mavenCentral()
 }
 
+sourceSets {
+    create("integrations") {
+        compileClasspath += sourceSets.main.get().output
+        runtimeClasspath += sourceSets.main.get().output
+    }
+}
+
+val integrationsImplementation: Configuration by configurations.getting {
+    extendsFrom(configurations.implementation.get())
+}
+
+configurations["integrationsRuntimeOnly"].extendsFrom(configurations.runtimeOnly.get())
+
```

Comment:
Not really certain as to the purpose of creating another method of importing packages.  Is there a reason why it can't be imported through testImplementation